src/components/App/App.tsx

```diff
@@ -7,8 +8,28 @@ import MainPage from '../MainPage/MainPage';
 import GamePage from '../GamePage/GamePage';
 import EndPage from '../EndPage/EndPage';
 import ErrorPage from '../ErrorPage/ErrorPage';
+import { Question } from '../../types';
+import { getQuestions } from '../../apiCall';
 
 function App() {
+
+  const [questions, setQuestions] = useState<Question | null>(null);
+
+  const fetchData = async () => {
```

Comment:
I like the use of async/await here and how the conditional is checking for the duplicate data coming in to handle a 429 error which we have been encountering.